server/src/main/java/org/opensearch/cluster/routing/remote/RemoteRoutingTableService.java

```diff
@@ -61,6 +64,15 @@ protected void doStart() {
         blobStoreRepository = (BlobStoreRepository) repository;
     }
 
+    public BlobStore getBlobStore() {
+        return blobStoreRepository.blobStore();
+    }
```

Comment:
Why do we need this method? We can directly use `blobStoreRepository` to get the blobStore in delete method.

Reply:
Removed in next revision.